UI/TechTreeWnd.cpp

```diff
@@ -564,6 +565,9 @@ class TechTreeWnd::LayoutPanel : public GG::Wnd {
     void TreeZoomedSlot(int move);
     void TreeZoomInClicked();
```

Comment:
In order to not add boost::lambda to the dependencies I left this as is.  When freeorion moves to C++11 it should be cleaned up with a lambda function to add the dummy true return value.